rust/tests/adls_gen2_backend_test.rs

```diff
@@ -1,39 +1,192 @@
 #[cfg(feature = "azure")]
+/// An Azure Data Lake Gen2 Storage Account is required to run these tests and must be provided by
```

Comment:
Could you share with me why the tests couldn't be run on the CI?

Reply:
Yes, please see here: #497